src/t8_schemes/t8_default/t8_default_hex/t8_default_hex_cxx.cxx

```diff
@@ -739,6 +739,14 @@ t8_default_scheme_hex_c::t8_element_is_valid (const t8_element_t *elem) const
   return p8est_quadrant_is_extended ((const p8est_quadrant_t *) elem)
     && T8_QUAD_GET_TDIM ((const p8est_quadrant_t *) elem) == 3;
 }
+
+void
+t8_default_scheme_hex_c::t8_element_debug_print (const t8_element_t *elem)
+{
+  p8est_quadrant_t   *quad = (p8est_quadrant_t *) elem;
+  t8_debugf ("x: %i, y: %i, z: %i, level: %i\n", quad->x, quad->y, quad->z,
+             quad->level);
```

Comment:
There is actually a p4est function for this `p4/8est_quadrant_print`. Since our element is a p4est_quadrant_t we should use the p4est function.